crates/tokmd-model/CLAUDE.md

```
@@ -0,0 +1,89 @@
+
+## Responsibility
+
+- Convert tokei `Languages` → tokmd receipts
+- Aggregation by language and module
+- Path normalization rules
+- Receipt generation
+- **NOT** for scanning or output formatting
+
+## Public API
+
+### Report Creation
```

markdownlint-cli2 (0.20.0)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)